src/api/v1/store/index.js

```diff
@@ -0,0 +1,337 @@
+const Sequelize = require('sequelize');
+const express = require('express');
+const { any, flatten, pick, intersection } = require('underscore');
+const email = require('../../../email');
+const error = require('../../../error');
+const { Activity, User, MerchandiseCollection, Merchandise, Order, OrderItem, db } = require('../../../db');
+
+const router = express.Router();
+
+router.route('/collection/:uuid?')
+  .get(async (req, res, next) => {
+    try {
+      if (req.params.uuid) {
+        const collection = await MerchandiseCollection.findByUUID(req.params.uuid).then((c) => (c ? c.get() : c));
+        if (!collection) return next(new error.NotFound('No such merchandise collection found'));
+        const merchandise = await Merchandise.findAllByCollection(collection.uuid, req.user.isAdmin());
+        collection.merchandise = req.user.isAdmin() ? merchandise : merchandise.map((m) => m.getPublicItem());
+        res.json({ error: null, collection });
+        return;
+      }
+      const findCollectionsMinusMerch = req.user.isAdmin()
+        ? MerchandiseCollection.getAllCollections() : MerchandiseCollection.getAllUnarchivedCollections();
+      const collectionsMinusMerch = (await findCollectionsMinusMerch).then((cs) => cs.map((c) => c.get()));
```

Comment:
Does this line cause an error for you? I'm getting a `TypeError: (intermediate value).then is not a function` on the `.then` when calling GET `/api/v1/store/collection/` as an admin (errors out as non-admin as well). I'm on the latest pull from this branch too.

Reply:
Fixed.